common/constant/key.go

```diff
@@ -313,11 +313,12 @@ const (
 const (
 	SUBSCRIBED_SERVICE_NAMES_KEY               = "subscribed-services"
 	PROVIDER_BY                                = "provided-by"
-	EXPORTED_SERVICES_REVISION_PROPERTY_NAME   = "dubbo.exported-services.revision"
+	EXPORTED_SERVICES_REVISION_PROPERTY_NAME   = "dubbo.metadata.revision"
```

Comment:
why not define another const name 'METADATA_REVISION_PROPERTY_NAME'?

Reply:
old EXPORTED_SERVICES_REVISION_PROPERTY_NAME not be used